### PR TITLE
-Z docs: Add link to unstable book

### PR DESCRIPTION
**Pull request by camsteffen**

### CODE REVIEW

File: src/doc/rustc/src/command-line-arguments.md [hunk 1]
```diff
@@ -302,7 +302,7 @@ _Note:_ The order of these lint level arguments is taken into account, see [lint
 This flag will allow you to set unstable options of rustc. In order to set multiple options,
 the -Z flag can be used multiple times. For example: `rustc -Z verbose -Z time-passes`.
 Specifying options with -Z is only available on nightly. To view all available options
-run: `rustc -Z help`.
+run: `rustc -Z help`, or see [The Unstable Book](https://doc.rust-lang.org/unstable-book/).
```
Comment (JohnTitor) [inline src/doc/rustc/src/command-line-arguments.md:305]: We prefer a relative link here:

```suggestion
run: `rustc -Z help`, or see [The Unstable Book](../unstable-book/).
```

Comment (camsteffen) [reply src/doc/rustc/src/command-line-arguments.md]: fixed

**Comment by JohnTitor** [comment]
Oh sorry, `index.html` (or `the-unstable-book.html`) is also needed. r=me once either is added and CI agrees with it.

**Review by JohnTitor — APPROVED** [review]

**Pull request merged**